OPHD/UI/Reports/FactoryReport.cpp

```diff
@@ -590,14 +586,14 @@ void FactoryReport::drawDetailPane(Renderer& renderer)
 	};
 	auto position = startPoint + NAS2D::Vector{138, 80};
 	for (auto [title, value] : requiredResources) {
-		drawTitleText(position, title, std::to_string(value), defaultTextColor);
+		drawLabelAndValueLeftJustify(position, title, std::to_string(value), labelWidth, defaultTextColor);
 		position.y() += 15;
 	}
 
 	// POPULATION
 	bool isPopulationRequirementHighlighted = SELECTED_FACTORY->populationAvailable()[0] != SELECTED_FACTORY->populationRequirements()[0];
 	auto text = std::to_string(SELECTED_FACTORY->populationAvailable()[0]) + " / " + std::to_string(SELECTED_FACTORY->populationRequirements()[0]);
-	drawTitleText(position, "Workers", text, (isPopulationRequirementHighlighted ? NAS2D::Color::Red : defaultTextColor));
+	drawLabelAndValueLeftJustify(position, "Workers", text, labelWidth, (isPopulationRequirementHighlighted ? NAS2D::Color::Red : defaultTextColor));
```

Comment:
Noting text justification was added here, so the change may need to be visually inspected.

Reply:
Actually, looking at this closer, the original solution was right justified labels, not left justified labels, so I messed up a lot of the calls in the FactoryReport. I'll look at it later today and create a new drawLabelAndValueRightJustify.

Thanks for pointing out.